examples/plot_knockoffs_wisconsin.py

```diff
@@ -0,0 +1,235 @@
+"""
+Controlled multiple variable selection on the Wisconsin breast cancer dataset
+=============================================================================
+
+In this example, we explore the basics of variable selection and illustrate the need to
+statistically control the amount of faslely selected variables. We compare two variable
+selection methods: the Lasso and the Model-X Knockoffs :footcite:t:`candes2018panning`.
+We show how the Lasso is not robust to the presence of irrelevant variables, while the
+Knockoffs (KO) method is able to address this issue.
+"""
+
+# %%
+import numpy as np
+import pandas as pd
+
+seed = 0
+rng = np.random.RandomState(seed)
+
+
+########################################################################################
+# Load the breast cancer dataset
+# ------------------------------------------------------
+# There are 569 samples and 30 features that correspond to tumor attributes.
+# The downstream task is to classify tumors as benign or malignant. We leave out 10% of
+# the data to evaluate the performance of the Lasso model on the prediction task.
```

Comment:
Note that it's not a lasso strictly speaking. Maybe call it "Logistic Lasso" or "sparse logistic".

Reply:
I tried to clarify this.